common/filestore_utils.py

```diff
@@ -27,34 +29,34 @@ def _using_gsutil():
     return experiment_filestore_path.startswith('gs://')
 
 
-if _using_gsutil():
-    from common import gsutil as filestore_utils_impl
-else:
+def get_impl():
+    """Returns the implementation for filestore_utils."""
+    if _using_gsutil():
+        return gsutil
     # Use local_filestore when not using gsutil.
-    # TODO(zhichengcai): Implement local_filestore.py and import it here.
-    from common import gsutil as filestore_utils_impl
+    return local_filestore
```

Comment:
Should we do the imports too here? (eg only import gsutil when actually needed)

Reply:
Don't think so, it's considered better to do imports at the toplevel (though our usage of the modules is probably not "ideal" either).